tests/models/git/test_modeling_git.py

```diff
@@ -495,3 +495,23 @@ def test_visual_question_answering(self):
         expected_shape = torch.Size((1, 15))
         self.assertEqual(generated_ids.shape, expected_shape)
         self.assertEquals(generated_caption, "what does the front of the bus say at the top? special")
+
+    def test_batched_generation(self):
+        processor = GitProcessor.from_pretrained("microsoft/git-base-coco")
+        model = GitForCausalLM.from_pretrained("microsoft/git-base-coco")
+        model.to(torch_device)
+
+        image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+        pixel_values = processor(images=image, return_tensors="pt").pixel_values
```

Comment:
would `images=[image, image]` work for creating a batch of size `2`? Not so important, just to make things shorter. If you decide to change, remember to move `# create batch of size 2` above this line.